databases.go

```diff
@@ -319,3 +319,19 @@ func (c *Client) DatabaseListUsers(ctx context.Context, app, addonID string) ([]
 func (c *Client) DatabaseDeleteUser(ctx context.Context, app, addonID, userName string) error {
 	return c.DBAPI(app, addonID).SubresourceDelete(ctx, "databases", addonID, "users", userName)
 }
+
+// DatabaseUserPasswordReset resets the password for a user for given database addon
+func (c *Client) DatabaseUserPasswordReset(ctx context.Context, app, addonID, username string) (DatabaseUser, error) {
```

Comment:
nitpick(non-blocking): why is the method named `...PasswordReset` whereas the endpoint is `reset_password`? I would have expected the method to be named `...ResetPassword`.

Reply:
Indeed... My automatism is to name hierarchically.
Database->User->Password->Reset
but here the action is "reset password".. I'll rename